SW_Control.c

```diff
@@ -46,6 +46,7 @@ extern SW_MODEL SW_Model;
 extern SW_VEGESTAB SW_VegEstab;
 extern SW_SITE SW_Site;
 extern SW_VEGPROD SW_VegProd;
+extern SW_FILE_STATUS Sw_File_Status;

```

Comment:
@BrendenBe1 Would you mind terribly to change the name of `Sw_File_Status` to match the naming scheme of every other variable in SOILWAT2? i.e., change it to `SW_File_Status`

Reply:
Ya I will make that change